test/integration/scheduler_perf/README.md

```diff
@@ -68,29 +68,22 @@ To produce a cpu profile:
 make test-integration WHAT=./test/integration/scheduler_perf KUBE_TIMEOUT="-timeout=3600s" KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling -cpuprofile ~/cpu-profile.out"
 ```
 
-### How to configure bechmark tests
+### How to configure benchmark tests
 
-Configuration file located under config/performance-config.yaml contains a list of templates.
+Configuration file located under `config/performance-config.yaml` contains a list of templates.
 Each template allows to set:
 - node manifest
 - manifests for initial and testing pod
 - number of nodes, number of initial and testing pods
 - templates for PVs and PVCs
 - feature gates
 
-See `simpleTestCases` data type implementation for available configuration for each template.
+See `op` data type implementation for available operations to compose WorkloadTemplates.
```

Comment:
nit: When I tried to find this `WorkloadTemplates`, I could not because the first character is lowercase `w` and `s` is not here in the code like `workloadTemplate` in performance-config.yaml.
How about making it the same as `workloadTemplate` or writing the path like config/performance-config.yaml ?
That can make it easy to understand, I feel.

Reply:
Thanks for pointing it out. As `op` refers to the source code in test/integration/scheduler_perf/scheduler_perf_test.go, so here I meant for `WorkloadTemplate`s. I think I need to back quota it.

Reply:
Thanks for your explanation, I got it.